libkbfs/folder_block_ops.go

```diff
@@ -1031,12 +1145,14 @@ func (fbo *folderBlockOps) ClearCachedDirEntry(lState *lockState, dir path) {
 	fbo.blockLock.Lock(lState)
 	defer fbo.blockLock.Unlock(lState)
 	delete(fbo.deCache, dir.tailPointer().Ref())
-
-	// TODO(KBFS-2076): delete the corresponding from the dirty block
-	// cache.  Only newly-created, empty dir blocks will be in the
-	// dirty block cache, but for now let's just try to delete all
-	// dirty dirs.  (In the future, modified directories should live
-	// in there as well.)
+	err := fbo.config.DirtyBlockCache().Delete(
+		fbo.id(), dir.tailPointer(), fbo.branch())
+	if err != nil {
+		// If delete can ever fail, we should plumb this error
```

Comment:
But `Delete` can't ever fail right now, so there's nothing to file a ticket about.  This comment is just speculating about some future world that doesn't exist.  Should probably just remove the `error` return param from `Delete` (in some other PR).